docs/building-images.md

```diff
@@ -91,70 +98,92 @@ operatingSystem:
+    * `servers` - Specifies a list of servers that NTP will use as data sources.
+* `proxy` - Defines system-wide proxy information.
+  * `httpProxy` - Sets the system-wide http proxy settings.
+  * `httpsProxy` - Sets the system-wide https proxy settings.
+  * `noProxy` - Overrides the default `NO_PROXY` list. By default, this is `localhost, 127.0.0.1` if this
+  parameter is omitted. If this option is set, the default entries will need to be manually added if they are
+  still in use.
+* `kernelArgs` - Provides a list of flags that should be passed to the kernel on boot.
+* `groups` - Defines a list of operating system groups to create. This will not fail if the 
+group already exists. Each entry is made up of the following fields:
   * `name` - Required; Name of the group to create.
   * `gid` - Optional; If specified, the group will be created with the given ID. If omitted, the GID will be generated
     by the operating system.
-* `users` - Optional; Defines a list of operating system users to be created. Each entry is made up of
+* `users` - Defines a list of operating system users to create. Each entry is made up of
   the following fields (one or both of the password and SSH key must be provided per user):
   * `username` - Required; Username of the user to create. To set the password or SSH key for the root user,
     use the value `root` for this field.
   * `uid` - Optional; If specified, the user will be created with the given ID. If omitted, the UID will be generated
     by the operating system.
   * `createHomeDir` - Optional; If set to `true`, a home directory will be created for the user. Defaults to `false`
-    if unspecified.
-  * `encryptedPassword` - Optional; Encrypted password to set for the use (for example, using `openssl passwd -6 $PASSWORD`
-    to generate the value for this field).
+  if unspecified. If one or more SSH keys is specified, this must be set to `true` to properly configure the
+  user.
+  * `encryptedPassword` - Optional; Encrypted password to set for the use (for example,
+  using `openssl passwd -6 $PASSWORD` to generate the value for this field).
   * `sshKeys` - Optional; List of public SSH keys to configure for the user.
-  * `primaryGroup` - Optional; If specified, the user will be configured with this as the primary group. The group
-    must already exist, either as a default group or one defined in the `groups` field. If this is omitted, the
-    result will be the default for the operating system (on SLE Micro, this is `users`).
+  * `primaryGroup` - Optional; If specified, the user will be configured with this value as the primary group. The group
+  must already exist, either as a default group or one defined in the `groups` field. If this is omitted, the
+  result will be the default for the operating system (on SLE Micro, this is `users`).
   * `secondaryGroups` - Optional; If specified, the user will be configured as part of each listed group. The
```

Comment:
These still have the `Optional;` prefix.

Reply:
Fixed